publication/ver11/accessibilty.md

```diff
@@ -0,0 +1,274 @@
+# Results of accessibility self-review for Web of Things (WoT) Architecture
+
+Answers to accessibility (FAST) review checklist [(from here)](https://w3c.github.io/apa/fast/checklist.html)
+for the Web of Things (WoT) architecture as a whole.
+These answers are generally written to be applicable to all WoT deliverables.
+Where a response is specifically relevant to a particular deliverable that will be identified.
+
+## If technology allows visual rendering of content
+The WoT architecture does not directly support visual rendering of content.
+Some devices adhering to WoT standards may have visual display aspects, for example
+light bulbs that can be turned on or off, or small displays.
+However, WoT in general has no provision to constrain visual displays, 
+as this (and other controls limiting the behavior of devices)
+was not in the scope of our current charter.
+
+### [no] There is a defined way for a non-visual rendering to be created.	
+No.
+### [no] Content can be resized.	
+No specified mechanism.
+### [possible] Luminosity and hue contrast can adapt to user requirements.	
```

Comment:
I would make it n/a. 
Light bulbs are only a very specific kind of device that was used in examples, but setting color or hue is not an intrinsic capability of WoT.

Reply:
Light bulbs are just an example of a device that can be used as a under interface.  In fact in the demo I just did for Conexxus I did exactly that, using color to indicate the state of a temperature sensor.  BUT I probably would want to provide controls to adjust the color.  For example, the original design use red, yellow, and green, but this has problems with red-green blindness, so I switched to red-white-blue.  I could also, in theory, map the color of the light (in a separate automation, by reading it out as a property) to an audible alert (that could announce when the color state changed, and to what).

Setting hue is not an *intrinsic* property of WoT but I can use WoT to *build systems* that support accessibility.

Reply:
anyway, to be consistent, as discussed, leaving it as "possible", but I did enhance the discussion of the points in this entire section